pkg/clients/ocm/oidc_test.go

```
@@ -0,0 +1,277 @@
+		URL(server.URL).
+		TokenURL(server.URL+"/token").
+		Client("test", "test").
+		TransportWrapper(func(_ http.RoundTripper) http.RoundTripper {
+			return server.Client().Transport
+		}).
+		Insecure(true).
+		Build()
+	if err != nil {
+		t.Fatalf("failed to build OCM connection: %v", err)
+	}
+	t.Cleanup(func() { conn.Close() })
```

golangci-lint (2.12.2)

[error] Error return value of `conn.Close` is not checked

(errcheck)